src/store/dashboard/sagas/dashboardBalances.sagas.ts

```diff
@@ -0,0 +1,113 @@
+import { BigNumber, constants } from 'ethers';
```

Comment:
rename the file to `fetchDashboardBalances.ts` to match our current convention. Same thing with `transactions.sagas`

Reply:
It would be cool if you could also move tests for these sagas from `dashboard.test.ts` to the `fetchDashboardBalances.test.ts` file